File: python-client-sdk/src/sdk/models/shared/projectrunstatus_enum.py
import dataclasses
from enum import Enum

class ProjectRunStatusEnum(str, Enum):
    PENDING = "PENDING"
    RUNNING = "RUNNING"
    ERRORED = "ERRORED"
    COMPLETED = "COMPLETED"
    KILLED = "KILLED"
    UNABLE_TO_ALLOCATE_KERNEL = "UNABLE_TO_ALLOCATE_KERNEL"

File: java-client-sdk/lib/src/main/java/models/shared/ProjectRunStatusEnum.java
package .models.shared;

import com.fasterxml.jackson.annotation.JsonValue;
public enum ProjectRunStatusEnum {
    PENDING("PENDING"),
    RUNNING("RUNNING"),
    ERRORED("ERRORED"),
    COMPLETED("COMPLETED"),
    KILLED("KILLED"),
    UNABLE_TO_ALLOCATE_KERNEL("UNABLE_TO_ALLOCATE_KERNEL");

    @JsonValue
    public final String value;

    private ProjectRunStatusEnum(String value) {
        this.value = value;
    }
}
